src/web-worker/web-worker.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Web worker entry point. Currently only used by a single
+ *   extension (amp-bind), so dependencies are directly imported.
+ *   Eventually, each extension that uses this worker will bundle its own
+ *   "lib" JS files and loaded at runtime via `importScripts()`.
+ */
+
+import '../../third_party/babel/custom-babel-helpers';
+import {BindEvaluator} from '../../extensions/amp-bind/0.1/bind-evaluator';
+import {FromWorkerMessageDef, ToWorkerMessageDef} from './web-worker-defines';
+
+/** @private {BindEvaluator} */
+let evaluator_;
+
+self.addEventListener('message', function(event) {
+  const {method, args, id} = /** @type {ToWorkerMessageDef} */ (event.data);
+
+  let returnValue;
+
+  // TODO(choumx): Add error reporting.
+  switch (method) {
```

Comment:
Let's wrap this in a try-catch, then post back an error if something goes wrong.